pandas/_typing.py

```diff
@@ -85,6 +85,20 @@
 PandasScalar = Union["Period", "Timestamp", "Timedelta", "Interval"]
 Scalar = Union[PythonScalar, PandasScalar]
 
+# numpy compatible types
+NumpyValueArrayLike = Union[PythonScalar, Sequence[PythonScalar]]
+NumpySorter = Union[
+    int,
+    np.integer,
+    bool,
+    np.ndarray,
+    Sequence[int],
+    Sequence[np.integer],
+    Sequence[bool],
+    None,
+]
```

Comment:
This is equivalent to `Optional[npt._ArrayLikeInt_co]`?

also use Optional[...] instead of Union[..., None]. https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#style-guidelines (Although the docs need updating for the new syntax, not relevant for aliases)

Reply:
Redefined as suggested in next commit.